app/data/events/get-event-order.js

```diff
@@ -26,8 +26,8 @@ const getEventOrder = (event) => {
     PAYMENT_PAUSED_QUALITY_CHECK,
     PAYMENT_QUALITY_CHECK_FAILED,
     PAYMENT_QUALITY_CHECK_PASSED,
-    PAYMENT_PROCESSED,
     PAYMENT_RESET,
+    PAYMENT_PROCESSED,
```

Comment:
Does the order of the const need updated too?

Reply:
No it's fine, this is the correct order now this event is added